lib/ClangImporter/ImportType.cpp

```diff
@@ -1287,10 +1287,24 @@ static ImportedType adjustTypeForConcreteImport(
   // we would prefer to instead use the default Swift convention.
```

Comment:
Sorry, I meant *this* comment, which refers to `@convention(block)`.

Reply:
Grrrr. That comment above is a lie. I'll do a separate comment-only PR for that